components/tokens/custom-spectrum/custom-medium-vars.css

```diff
@@ -21,4 +21,10 @@ governing permissions and limitations under the License.
   
   --spectrum-slider-tick-mark-height: 10px;
   --spectrum-slider-ramp-track-height: 16px;
+
+  --spectrum-colorwheel-path: "M 95 95 m -95 0 a 95 95 0 1 0 190 0 a 95 95 0 1 0 -190 0.2 M 95 95 m -73 0 a 73 73 0 1 0 146 0 a 73 73 0 1 0 -146 0";
+  --spectrum-colorwheel-path-borders: "M 96 96 m -96 0 a 96 96 0 1 0 192 0 a 96 96 0 1 0 -192 0.2 M 96 96 m -72 0 a 72 72 0 1 0 144 0 a 72 72 0 1 0 -144 0";
+  --spectrum-colorwheel-colorarea-container-size: 144px;
+
+  --spectrum-colorhandle-size: 16px;
```

Comment:
same comment as the large, should we add a TODO comment here for the colorhandle-size token?

Reply:
Yep! Done.